examples/projections/table/README.txt

```diff
@@ -19,64 +19,85 @@ The below table shows the projection codes for the 31 GMT projections.
-     - Lambert conic conformal
+     - :ref:`Lambert conic conformal
+       <sphx_glr_projections_conic_conic_lambert.py>`
    * - **M**\ [|lon0|\ [/|lat0|]/]\ *width*
-     - Mercator cylindrical
+     - :ref:`Mercator cylindrical <sphx_glr_projections_cyl_cyl_mercator.py>`
    * - **N**\ [|lon0|/]\ *width*
-     - Robinson
+     - :ref:`Robinson <sphx_glr_projections_misc_misc_robinson.py>`
    * - **Oa**\ |lon0|/|lat0|/\ *azim*/*width*\ [**+v**]
      - Oblique Mercator, 1: origin and azim
    * - **Ob**\ |lon0|/|lat0|/|lon1|/|lat1|/\ *width*\ [**+v**]
      - Oblique Mercator, 2: two points
    * - **Oc**\ |lon0|/|lat0|/|lonp|/|latp|/\ *width*\ [**+v**]
      - Oblique Mercator, 3: origin and pole
-   * - **P**\ *width*\ [**+a**]\ [**+f**\ [**e**\|\ **p**\|\ *radius*]][**+r**\ *offset*][**+t**\ *origin*][**+z**\ [**p**\|\ *radius*]]
-     - Polar [azimuthal] (:math:`\theta, r`) (or cylindrical)
+   * - **P**\ *width*\ [**+a**]\ [**+f**\ [**e**\|\ **p**\|\ *radius*]]\
+       [**+r**\ *offset*][**+t**\ *origin*][**+z**\ [**p**\|\ *radius*]]
+     - :ref:`Polar <sphx_glr_projections_nongeo_polar.py>` [azimuthal]
+       (:math:`\theta, r`) (or cylindrical)
    * - **Poly**\ [|lon0|\ [/|lat0|]/]\ *width*
-     - Polyconic
+     - :ref:`Polyconic <sphx_glr_projections_conic_polyconic.py>`
    * - **Q**\ [|lon0|\ [/|lat0|/]]\ *width*
-     - Equidistant cylindrical
+     - :ref:`Equidistant cylindrical
+       <sphx_glr_projections_cyl_cyl_equidistant.py>`
    * - **R**\ [|lon0|/]\ *width*
-     - Winkel Tripel
+     - :ref:`Winkel Tripel <sphx_glr_projections_misc_misc_winkel_tripel.py>`
    * - **S**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*
-     - General stereographic
+     - :ref:`General stereographic
+       <sphx_glr_projections_azim_azim_general_stereographic.py>`
    * - **T**\ [|lon0|\ [/|lat0|]/]\ *width*
-     - Transverse Mercator
+     - :ref:`Transverse Mercator
+       <sphx_glr_projections_cyl_cyl_transverse_mercator.py>`
    * - **U**\ *zone*/*width*
-     - Universal Transverse Mercator (UTM)
+     - :ref:`Universal Transverse Mercator (UTM)
+       <sphx_glr_projections_cyl_cyl_universal_transverse_mercator.py>`
    * - **V**\ [|lon0|/]\ *width*
-     - Van der Grinten
+     - :ref:`Van der Grinten
+       <sphx_glr_projections_misc_misc_van_der_grinten.py>`
    * - **W**\ [|lon0|/]\ *width*
-     - Mollweide
-   * - **X**\ *width*\ [**l**\|\ **p**\ *exp*\|\ **T**\|\ **t**][/\ *height*\ [**l**\|\ **p**\ *exp*\|\ **T**\|\ **t**]][**d**]
-     - Linear, log\ :math:`_{10}`, :math:`x^a-y^b`, and time
+     - :ref:`Mollweide <sphx_glr_projections_misc_misc_mollweide.py>`
+   * - **X**\ *width*\ [**l**\|\ **p**\ *exp*\|\ **T**\|\ **t**][/\ *height*\
+       [**l**\|\ **p**\ *exp*\|\ **T**\|\ **t**]][**d**]
+     - :ref:`Linear <sphx_glr_projections_nongeo_cartesian_linear.py>`,
+       :ref:`log <sphx_glr_projections_nongeo_cartesian_logarithmic.py>`\
+       :math:`_{10}`, :ref:`power
```

Comment:
```suggestion
       :math:`_{10}`, :ref:`power
```
I guess it's not possible to include the subscripted **10**?

Reply:
I do not know of a way to get that to work.

Reply:
Hmm, of course it's only a minor issue but it would look better if we could fix that in any way.

Reply:
I changed it to `logarithmic`